### PR TITLE
DR-369 Add action to run bootstrap

### PR DESCRIPTION
**Pull request by timrickwood**
## Description

After the first manually-triggered run of the bootstrapper script, any repeat run of the script can be triggered automatically via a pipeline. This ensures the infrastructure created/maintained by the bootstrapper script is kept in line with IAC without additional manual steps
## Context

Removes the need to re-run bootstrapper script manually and saves time/effort and reduces risks inherent in manual running
## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [x ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: .github/workflows/orchestrate-bootstrap.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Orchestrate bootstrapper workflow
```
Comment (tamaragoldschmidt) [inline .github/workflows/orchestrate-bootstrap.yaml:1]: is that a zero or a capital o ?

Comment (timrickwood) [reply .github/workflows/orchestrate-bootstrap.yaml:1]: A capital o(O) (in my VSC code at least a 0 has a line through it to denote the difference)

File: .github/workflows/orchestrate-bootstrap.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+    re-run-bootstrap-dos-dev:
```
[the remark this answers is not in the document]
Comment (timrickwood) [reply .github/workflows/orchestrate-bootstrap.yaml:9]: Interesting. I deliberately went with re-run to show/imply that it has to have been run before (manually) but if we don't think that is helpful I can change it to run-bootstrap-etc

[the remark this answers is not in the document]
Comment (jope1) [reply .github/workflows/orchestrate-bootstrap.yaml:9]: run_bootstrap feels better

**Review by jope1 — APPROVED** [review]

File: .github/workflows/orchestrate-bootstrap.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (timrickwood) [reply .github/workflows/orchestrate-bootstrap.yaml]: Changing to run-bootstrap-dos-dev etc - ie dropping the re-

**Review by tamaragoldschmidt — APPROVED** [review]

**Pull request merged**